### PR TITLE
FixedBitVector: Fix container value

### PR DESCRIPTION
**Pull request by obastemur**
/cc @LouisLaf

### CODE REVIEW

**Comment by obastemur** [comment]
@dotnet-bot test Ubuntu ubuntu_linux_debug please

File: lib/Common/DataStructures/FixedBitVector.h [hunk 1]
```diff
@@ -241,7 +241,20 @@ void BVFixed::SetRange(Container* value, BVIndex start, BVIndex len)
     BVIndex iEnd = BVUnit::Position(end);
     BVIndex oStart = BVUnit::Offset(start);
     BVIndex oEnd = BVUnit::Offset(end);
-    BVUnit::BVUnitTContainer* bits = (BVUnit::BVUnitTContainer*)value;
+
+    BVUnit::BVUnitTContainer temp;
+    BVUnit::BVUnitTContainer* bits;
+    static_assert(sizeof(Container) == 1 || sizeof(Container) == sizeof(BVUnit::BVUnitTContainer),
+        "Container is not suitable to represent the calculated value");
+    if (sizeof(BVUnit::BVUnitTContainer) == 1)
```
Comment (obastemur) [inline lib/Common/DataStructures/FixedBitVector.h]: notes to reviewer: this should be optimized away on final declaration.

**Comment by LouisLaf** [comment]
:shipit:

**Comment by obastemur** [comment]
@LouisLaf Thanks for the review

**Pull request merged**